app/controllers/sessions_controller.rb

```diff
@@ -14,8 +14,8 @@ def create
       flash[:success] = {color: 'green', message: "Logged in as #{@user.first_name}"}
       redirect_to dashboard_path
     else
-      flash[:error] = {color: 'orange', message: "Username or Password incorrect."}
-      render :new
+      flash[:error] = {color: 'orange', message: "Username or password incorrect."}
```

Comment:
Line is too long. [83/80]<br>Space inside { missing.<br>Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.